src/main/java/org/frc5687/powerup/robot/Robot.java

```diff
@@ -226,6 +227,10 @@ protected void loopFunc() {
         }
     }
 
+    public void poll(){
```

Comment:
We're not going to use a poll method in Robot for anything else, so just call oi.poll() directly in teleopPeriodic()